airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -166,10 +171,30 @@ public void tearDown() throws ApiException, SQLException {
     }
   }
 
+  @Test
+  @Order(-1)
```

Comment:
is `@Order(-1)` kosher?

Reply:
yes I believe so, the docs didn't say anything about the number needing to be positive. in hindsight it's probably better to separate these out by 10 and start them at something like 100 to avoid weird numbers like this. NBD atm.